C-Plus-Plus/other/tower_of_hanoi.cpp

```diff
@@ -0,0 +1,32 @@
+// C++ recursive function to
+// solve tower of hanoi puzzle
+#include <bits/stdc++.h>
+using namespace std;
+
+void towerOfHanoi(int n, char from_rod,
+                  char to_rod, char aux_rod)
+{
+    if (n == 1)
+    {
+        cout << "Move disk 1 from rod " << from_rod <<
+             " to rod " << to_rod << endl;
+        return;
+    }
+    towerOfHanoi(n - 1, from_rod, aux_rod, to_rod);
+    cout << "Move disk " << n << " from rod " << from_rod <<
+         " to rod " << to_rod << endl;
+    towerOfHanoi(n - 1, aux_rod, to_rod, from_rod);
+}
+
+// Driver code
+int main()
+{
+    int n; // Number of disks
+    cout << "Enter number of disks: ";
+    cin >> n;
+    
+    towerOfHanoi(n, 'A', 'C', 'B'); // A, B and C are names of rods
+  
+    cout << "All disks have been moved to another rod!!!";
+    return 0;
+}

```

Comment:
Add the sample input-output in comment

Reply:
I have done that inside the function.

Reply:
keep that comment outside the code in the bottom of the source code

Reply:
Done